code/main.py

```diff
@@ -0,0 +1,285 @@
+import Featurizer
+import tkinter as tk
+from tkinter import ttk
+from tkinter import filedialog
+from tkinter import messagebox
```

Comment:
Is there a reason to not just use `tk.filedialog` and `tk.messagebox`?

Reply:
I tried that at first and it did not work. After doing some research, I found out that both filedialog and messagebox are not included in the list of ```__all__``` of tkinter and that separate imports are required for those modules. They say that that's simply how the package was designed. Looking at how other more advanced tkinter users have done it, separate imports like so seem to be the standard with tkinter (I think).

Reply:
Ok, fair enough!